config/build.conf.js

```diff
@@ -151,10 +161,20 @@ module.exports = {
     },
     metadata: {
```

Comment:
This may need to be restructured a bit for actually releasing these artifacts, but for now this works.